zgw/zaken-api/src/main/kotlin/nl/nlportal/zakenapi/service/ZakenApiService.kt

```diff
@@ -44,13 +44,17 @@ class ZakenApiService(
         page: Int,
         authentication: CommonGroundAuthentication,
         zaakTypeUrl: String?,
+        isOpen: Boolean,
     ): ZaakPage {
         val request =
             zakenApiClient.zaken()
                 .search()
                 .withAuthentication(authentication)
 
         zaakTypeUrl?.let { request.ofZaakType(it) }
+        if (isOpen) {
```

Comment:
Is isOpen niet optioneel?
Is het dan niet beter om isOpen nullable te maken en alleen als isOpen een waarde heeft deze aan de request toe te voegen?

Reply:
helemaal correct, ga het aanpassen